shaders/src/shading_lit.fs

```diff
@@ -72,7 +72,21 @@ void getCommonPixelParams(const MaterialInputs material, inout PixelParams pixel
     // Assumes an interface from air to an IOR of 1.5 for dielectrics
     float reflectance = computeDielectricF0(material.reflectance);
 #endif
+#if !defined(MATERIAL_HAS_SPECULAR_FACTOR) && !defined(MATERIAL_HAS_SPECULAR_COLOR_FACTOR)
     pixel.f0 = computeF0(baseColor, material.metallic, reflectance);
+#else
+    vec3 dielectricSpecularF0 = vec3(0.0);
+    float dielectricSpecularF90 = 0.0;
+#if defined(MATERIAL_HAS_SPECULAR_COLOR_FACTOR)
+    dielectricSpecularF0 = min(0.04 * material.specularColorFactor, vec3(1.0));
```

Comment:
When the extension `KHR_materials_ior` is used, 0.04 must be replaced with the value computed from the IOR. See https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_materials_specular/README.md#interaction-with-other-extensions

Reply:
0.04 also needs to be changed when `reflectance` is set

Reply:
seems like reflectance is a good replace of 0.04 since it is default to 0.04 and calculated correctly in IOR case.